index.html

```diff
@@ -0,0 +1,147 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <link rel="icon" type="image/svg+xml" href="favicon.svg" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Tech Academy</title>
+  <link rel="stylesheet" href="https://use.typekit.net/kjf8squ.css">
+  <script src="https://kit.fontawesome.com/67c66657c7.js"></script>
+  
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <header class="primary-header">
+    <div class="container">
+          <nav class="navigation">
+            <div class="menu-button"><img id="humberger" src="img/menu.svg" alt="menu-button"></div>
+            <a href="index.html"><img class="logo"src="img/logo-2.png" alt="Manage" /></a>
+          <div class="nav-bar">
+          <ul  class="nav-list text-neutral-900 fw-bold fs-500" >
+          <li><a class="nav-link"href="index.html">Home</a></li>
+          <li><a class="nav-link" href="about-page.html">About</a></li>
+          <li><a class="nav-link" href="#courses">Courses</a></li>
+          <li><a class="nav-link" href="#instructor">Instrictors</a></li>
+        </ul>
+          </div>
+        
+      </nav>
+    </div>
+  </header>
+  <div id="nav-wrap">
+    <div id="close-button"><img src="img/x-icon.svg" alt="close-button"></div>
+    <div id="nav-m-wrap">
+      <li><a href="index.html">Home</a></li>
+      <li><a href="about-page.html">About</a></li>
+      <li><a href="#courses">Courses</a></li>
+      <li><a href="#instructor">Instrictors</a></li>
+    </div>
+  </div>
```

Comment:
- [ ] You have made this code for mobile menu and you have made separate code for your desktop nav. This is a very unprofessional way of doing it because suppose if tomorrow you will expand this website for multiple resolutions devices then you will have to increase the code which will make your application slow and difficult to manage. Kindly use the same code for mobile menu which you have made for the desktop menu :+1: and make your navbar for mobile using css instead of increasing code in html :+1: